docs/dashboards/visualization/add.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: default
+title: Create a visualization
+nav_order: 1
+parent: Visualizations
+grand_parent: Dashboards
+has_children: false
+has_toc: false
+---
+
+# How to create a visualization ?
+
+In order to make this section easy to understand, we'll divide into 3
+sub-sections:
+- Choosing a visualization type
+- Choosing an index
+- Making the visualization
+
+## Choosing a visualization type
+
+A visualization, as the name suggests, is the representation of your data in the
+form of a diagram of some sort (pie-chart, data table, heat map etc...)
+
+In order to start creating your visualization, you need to decide on a
+visualization type. Usually the choice of a visualization type is an obvious
+choice since the user will have an idea of what it should look like.
+
+### Steps
+1. Navigate to the `Visualize` dashboard. You can do so by clicking on the
+   `Visualize` icon in the sidebar. It's the second icon from the top.
+2. Once you have the above open, click on the `+` icon to open up the
+   visualization types panel. ![visualization
```

Comment:
The image link is broken

Do you think it is better to move viz assets to viz folder to avoid such errors?